src/popup/popup.html

```diff
@@ -204,6 +204,18 @@ <h6 class="vocab heading info__label margin-bottom-medium">Share</h6>
         </div>
       </div>
     </div>
+  <div id="myModal" class="modal">
+        <!-- Modal content -->
+        <div class="modal-content">
+       <h3 class="popup-header">Alert</h3>
+<span class="modal-close">&#10006;</span>
+          <div class="popup-text" id="modal-text">
+            <p align="centre"></p>
```

Comment:
What is the use of this (`<p align="centre"></p>`)? In your implementation, this paragraph is overwritten by using`innerText` and therefore the content is not centered.